samcli/lib/providers/sam_api_provider.py

```diff
@@ -299,7 +299,7 @@ def _extract_authorizers_from_props(logical_id: str, auth: dict, collector: ApiC
         authorizers: Dict[str, Authorizer] = {}
 
         for auth_name, auth_props in auth.get(SamApiProvider._AUTHORIZERS, {}).items():
-            authorizer_type = auth_props.get(SamApiProvider._FUNCTION_TYPE, LambdaAuthorizer.TOKEN)
+            authorizer_type = auth_props.get(SamApiProvider._FUNCTION_TYPE, LambdaAuthorizer.TOKEN.upper())
```

Comment:
Is this constant used in other places where it needs to be lowercase?

Reply:
same comment as daniel other than that LGTM

Reply:
Nope, `authorizer_type` is only used to check if the authorizer can be extracted. There isn't any other occurrence in that function.